### ISSUE TITLE
Create Benchmark on Gitbook

### ISSUE THREAD

**Issue by Marco-Pellegrino** [Alpaca4d/Alpaca4d#21]
Testing needs to be done and it is also a great opportuni to put material online.

Create a Benchmark Section on Gitbook

**Comment by Marco-Pellegrino** [comment]
@Goniometrie 
The idea is to take some common case and show to the user that Alpaca4d is correctly computing the solution.

https://alpaca4d.gitbook.io/docs/benchmark/intro

Feel free to add section in "benchmark"

**Comment by Marco-Pellegrino** [comment]
@Goniometrie 
Should we write down a list?

- [ ] simply supported beam
- [ ] cantiliver beam
- [ ] shell plate ?
- [ ] brick beam ?

**Comment by Goniometrie** [comment]
Yes that is a good start. From there we can add more complex benchmarks that can still be validated by analytical equations
